src/Tizen.NUI/src/public/Common/NUIConstants.cs

```diff
@@ -1829,6 +1829,83 @@ public struct FontSizeScale
         public const float UseSystemSetting = -1.0f;
     }
 
+    /// <summary>
+    /// Offset has left, right, bottom, top value.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct Offset
+    {
+        private int left;
+        private int right;
+        private int bottom;
+        private int top;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="left">left offset</param>
+        /// <param name="right">right offset</param>
+        /// <param name="bottom">bottom offset</param>
+        /// <param name="top">top offset</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Offset(int left, int right, int bottom, int top)
+        {
+            this.left = left;
+            this.right = right;
+            this.bottom = bottom;
+            this.top = top;
+        }
+
+        /// <summary>
+        /// Left
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int Left
+        {
+            get
+            {
+                return left;
+            }
+        }
```

Comment:
You can use auto generated getter on property without field
```suggestion
        /// <summary>
        /// Constructor
        /// </summary>
        /// <param name="left">left offset</param>
        /// <param name="right">right offset</param>
        /// <param name="bottom">bottom offset</param>
        /// <param name="top">top offset</param>
        [EditorBrowsable(EditorBrowsableState.Never)]
        public Offset(int left, int right, int bottom, int top)
        {
            Left = left;
            Right = right;
            Bottom = bottom;
            Top = top;
        }

        /// <summary>
        /// Left
        /// </summary>
        [EditorBrowsable(EditorBrowsableState.Never)]
        public int Left {get;}
```